### DISCUSSION TITLE
Vercel Security Checkpoint

### DISCUSSION THREAD

**Discussion by Saypanker** [vercel/community#7167] [Help] [answered]
### Summary

This problem has appeared; this has never happened before. Now some sites won't let me go anywhere. I use Chrome, FireFox, Opera, Microsoft Edge. What should I do in this situation? How can I solve the problem?

### Steps to Reproduce

This problem has appeared; this has never happened before. Now some sites won't let me go anywhere. I use Chrome, FireFox, Opera, Microsoft Edge. What should I do in this situation? How can I solve the problem?

**Comment by versecafe** [comment]
What is your problem?

**Comment by amyegan** [comment, marked as answer]
It sounds like you've visited some sites that have enabled [Attack Challenge Mode](https://vercel.com/docs/security/attack-challenge-mode). This adds an extra layer of protection to sites during attacks while allowing legitimate site visitors access to the site with a slight slowdown to get through the security checkpoint.

Or it's possible that these sites have used the Vercel Web Application Firewall to block your access with [WAF IP Blocking](https://vercel.com/docs/security/vercel-waf/ip-blocking#waf-ip-blocking) or [Web Application Firewall Custom Rules](https://vercel.com/docs/security/vercel-waf/custom-rules#waf-custom-rules).

[a comment by an automated account is not shown]
